### ISSUE TITLE
spec2bids run record not reproducible

### ISSUE THREAD

**Issue by mih** [psychoinformatics-de/datalad-hirni#12]
This is how it looks like. Lots of values are absolute paths. We can only use reliative paths
```
    [DATALAD RUNCMD] DICOM conversion of session /home/data/psyinf/scratch/multires3t/bids/sourcedata/REDACTED.
    
    === Do not change lines below ===
    {
     "outputs": [
      "/home/data/psyinf/scratch/multires3t/bids"
     ],
     "exit": 0,
     "cmd": [
      "singularity",
      "exec",
      "--bind",
      "/home/data/psyinf/scratch/multires3t/bids",
      ".datalad/environments/conversion/image",
      "heudiconv",
      "-f",
      "/home/mih/datalad-hirni/datalad_hirni/support/hirni_heuristic.py",
      "-s",
      "REDACTED",
      "-c",
      "dcm2niix",
      "-o",
      "/home/data/psyinf/scratch/multires3t/bids/.git/stupid/REDACTED",
      "-b",
      "-a",
      "/home/data/psyinf/scratch/multires3t/bids",
      "-l",
      "",
      "--minmeta",
      "--files",
      "/home/data/psyinf/scratch/multires3t/bids/sourcedata/REDACTED/dicoms"
     ],
     "pwd": ".",
     "dsid": "0470a5e4-625f-11e8-b78c-a0369f7c647e",
     "inputs": [
      "/home/data/psyinf/scratch/multires3t/bids/sourcedata/REDACTED/dicoms",
      "/home/data/psyinf/scratch/multires3t/bids/sourcedata/REDACTED/studyspec.json",
      ".datalad/environments/conversion/image"
     ],
     "chain": []

```

See https://github.com/datalad/datalad-container/issues/34 for one side-aspect of this situation.

**Comment by mih** [comment]
We are almost there:

```
singularity \
  exec --bind {pwd} .datalad/environments/conversion/image heudiconv \
  -f /home/mih/hacking/hirni/datalad_hirni/support/hirni_heuristic.py \
  -s XX -c dcm2niix \
  -o .git/hirni-tmp-35p3dmcq -b \
  -a /home/data/psyinf/scratch/multires3t/bids -l '' --minmeta \
  --files sourcedata/XXX/dicoms",
```

The argument of `-a` can become '{{pwd}}'.

That leaves only the location of the heuristic.

**Comment by mih** [comment]
> The argument of -a can become '{{pwd}}'.

Actually: `{dspath}`: https://github.com/datalad/datalad/pull/2632

**Comment by mih** [comment]
Solved except for #56, which cannot really be addressed here.
